docscan/session/create/filter/orthogonal_restrictions_filter.go

```diff
@@ -0,0 +1,65 @@
+package filter
+
+// RequestedOrthogonalRestrictionsFilter filters for a required document, allowing specification of restrictive parameters
+type RequestedOrthogonalRestrictionsFilter struct {
+	CountryRestriction *CountryRestriction `json:"country_restriction"`
+	TypeRestriction    *TypeRestriction    `json:"type_restriction"`
+}
+
+// RequestedOrthogonalRestrictionsFilterBuilder builds a RequestedOrthogonalRestrictionsFilter
+type RequestedOrthogonalRestrictionsFilterBuilder struct {
+	countryRestriction *CountryRestriction
+	typeRestriction    *TypeRestriction
+}
+
+// NewRequestedOrthogonalRestrictionsFilterBuilder creates a new RequestedOrthogonalRestrictionsFilterBuilder
+func NewRequestedOrthogonalRestrictionsFilterBuilder() *RequestedOrthogonalRestrictionsFilterBuilder {
+	return &RequestedOrthogonalRestrictionsFilterBuilder{
+		countryRestriction: &CountryRestriction{},
+		typeRestriction:    &TypeRestriction{},
+	}
+}
+
+// WithIncludeCountryList sets an "INCLUDE" list of country codes on the filter
+func (b *RequestedOrthogonalRestrictionsFilterBuilder) WithIncludeCountryList(countryCodes []string) *RequestedOrthogonalRestrictionsFilterBuilder {
```

Comment:
I think one of these might read better

```suggestion
func (b *RequestedOrthogonalRestrictionsFilterBuilder) WithIncludedCountries(countryCodes []string) *RequestedOrthogonalRestrictionsFilterBuilder {
```

```suggestion
func (b *RequestedOrthogonalRestrictionsFilterBuilder) WithIncludedCountryCodes(countryCodes []string) *RequestedOrthogonalRestrictionsFilterBuilder {
```